packages/strapi-parts/src/Combobox/components.js

```diff
@@ -18,13 +19,27 @@ export const MainRow = styled(Row)`
   `
       : undefined}
 
-  &:focus-within {
-    outline: none;
-    box-shadow: 0 0 0 2px rgba(66, 153, 225, 0.6);
-  }
+  ${inputFocusStyle()}
+`;
+
+export const ValueContainer = styled.div`
+  padding: 1px 2px;
+  grid-area: 1 / 1 / 2 / 3;
+`;
+
+export const InputContainer = styled.div`
```

Comment:
You should be able to use Row and the new props on box instead of overriding this :)